vdp/pipeline/v1beta/connector_definition.proto

```diff
@@ -80,7 +83,8 @@ message ConnectorDefinition {
   string title = 4 [(google.api.field_behavior) = OUTPUT_ONLY];
   // Connector definition documentation URL.
   string documentation_url = 5 [(google.api.field_behavior) = OUTPUT_ONLY];
-  // Connector definition icon.
+  // Connector definition icon. This is a relative path that allows clients
```

Comment:
ℹ️ We agreed that the icons would be moved to the `connector` and `operator` repos. Until that happens, the source of truth for the icon is `console`.